packages/scss/src/utilities/_text.scss

```diff
@@ -16,14 +16,14 @@
 
 // COLORS
 // ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
-@mixin textPalette($palette) {
+@mixin textColoring($palette) {
 	color: _get($palette, "color") !important;
 }
 @each $name, $palette in _palettes() {
-	.u-text-#{$name} {
-		@include textPalette($palette);
+	.u-text#{str-capitalize($name)} {
+		@include textColoring($palette);
```

Comment:
Couldn't we use the content of the mixin rather than having a 1-line mixin used 1 time ?